onedrive.service.in

```diff
@@ -4,7 +4,7 @@ Documentation=https://github.com/skilion/onedrive
 
 [Service]
 ExecStart=@PREFIX@/bin/onedrive -m
-Restart=no
+Restart=always
```

Comment:
Maybe use `Restart=on-failure` instead, so a clean exit doesn't restart the daemon?